cpp/src/aztec/plonk/proof_system/prover/c_bind.cpp

```diff
@@ -30,97 +30,100 @@ WASM_EXPORT void* test_async_func(size_t size, int val)
     }
 }
 
-WASM_EXPORT void prover_process_queue(waffle::TurboProver* prover)
+typedef std::conditional_t<waffle::SYSTEM_COMPOSER == waffle::TURBO, waffle::TurboProver, waffle::UltraProver>
+    WasmProver;
+
+WASM_EXPORT void prover_process_queue(WasmProver* prover)
 {
     prover->queue.process_queue();
 }
 
-WASM_EXPORT size_t prover_get_circuit_size(waffle::TurboProver* prover)
+WASM_EXPORT size_t prover_get_circuit_size(WasmProver* prover)
```

Comment:
Longer term, we want to use _only_ the unrolled prover, so we should get rid of the linearised prover to reduce some code complexity.